.circleci/config.yml

```diff
@@ -262,24 +262,80 @@ jobs:
           name: Build production code
           command: make build
           working_directory: ~/project/messaging/relayer
+
+  relayer-test:
+    machine:
+      image: ubuntu-2004:202107-02
```

Comment:
should we instead use the latest machine image `ubuntu-2004:202111-01`? [It seems to already have Go 1.17.3 installed](https://discuss.circleci.com/t/linux-machine-executor-images-2021-november-q4-update/42005), which might remove the need for the upgrade step below

Reply:
Good point. Done.